docs/userguide.rst

```diff
@@ -10,9 +10,9 @@ REANA cluster uses `Kubernetes <https://kubernetes.io/>`_ container
 orchestration system. The best way to try it out locally on your laptop is to
 install:
 
-- ``kubectl`` 1.14 or higher (see `kubectl installation guide <https://kubernetes.io/docs/tasks/tools/install-kubectl/>`_)
```

Comment:
Looks good, but two things:

(1) Please amend also those lines in user guide that speak of 1.14:

```console
$ git grep '1\.14' docs
docs/developerguide.rst:156:     version: "v1.14.0"
docs/developerguide.rst:476:    Starting local Kubernetes v1.14.0 cluster...
docs/developerguide.rst:539:      version: "v1.14.0"
docs/developerguide.rst:592:    Starting local Kubernetes v1.14.0 cluster...
docs/gettingstarted.rst:194:        version: "v1.14.0"
docs/userguide.rst:148:   Starting local Kubernetes v1.14.0 cluster...
```

(2) What about min/max versions?

```console
$ git grep '1\.14' reana_cluster
reana_cluster/backends/kubernetes/k8s.py:43:        'min_version': 'v1.14.0',
reana_cluster/backends/kubernetes/k8s.py:44:        'max_version': 'v1.14.0',
reana_cluster/configurations/reana-cluster-minikube-dev.yaml:3:  version: "v1.14.0"
reana_cluster/configurations/reana-cluster-minikube.yaml:3:  version: "v1.14.0"
```